test/src/test/java/org/axonframework/test/FixtureTest_RegularParams.java

```diff
@@ -341,19 +337,22 @@ public void testFixture_WrongEventContents() {
                                             new MyEvent("aggregateId", 3));
         MyCommandHandler commandHandler = new MyCommandHandler(fixture.getRepository(),
                                                                fixture.getEventBus());
-        try {
+
+        Throwable throwable = catchThrowable(() ->
             fixture
-                    .registerAnnotatedCommandHandler(commandHandler)
-                    .given(givenEvents)
-                    .when(new TestCommand("aggregateId"))
-                    .expectEvents(new MyEvent("aggregateId", 5)) // should be 4
-                    .expectVoidReturnType();
-            fail("Expected an AxonAssertionError");
-        } catch (AxonAssertionError e) {
-            assertTrue(e.getMessage().contains(
-                    "In an event of type [MyEvent], the property [someValue] was not as expected."));
-            assertTrue(e.getMessage().contains("Expected <5> but got <4>"));
-        }
+                .registerAnnotatedCommandHandler(commandHandler)
+                .given(givenEvents)
+                .when(new TestCommand("aggregateId"))
+                .expectEvents(new MyEvent("aggregateId", 5)) // should be 4
+                .expectVoidReturnType()
+        );
+
+        assertThat(throwable).isExactlyInstanceOf(AxonAssertionError.class)
+            .hasMessageContainingAll(
+                "field/property 'someValue' differ",
```

Comment:
The messages from those errors changed significantly. Don't get me wrong, they're still readable but became a lot bigger. If we apply this in CIM, our logs might become too big, since one of these exceptions occurs every now and then (aka. `ThatsOkException` 😉 ).